### PR TITLE
Add types definitions for @krakenjs/belter 

### PR DESCRIPTION
**Pull request by SlyBouhafs**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

---

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by Xenfo** [comment]
@SlyBouhafs the type definitions seem incomplete based on only [this ](https://github.com/krakenjs/belter/tree/master/test/tests/device) folder I found in Belter's tests. I would recommend adding all definitions.

**Comment by SlyBouhafs** [comment]
Should be good now, i added all the definitions for the device file.

File: types/belter/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Type definitions for belter 1.0
+// Project: https://github.com/krakenjs/belter#readme
+// Definitions by: sly <https://github.com/SlyBouhafs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Device.js function definitions
```
Comment (peterblazejewicz) [inline types/belter/index.d.ts:6]:
```suggestion
export as namespace belter;

// Device.js function definitions
```
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
https://unpkg.com/belter@1.0.142/dist/belter.js

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@SlyBouhafs thx!

**Comment by SlyBouhafs** [comment]
Ready to merge

**Pull request merged**